### ISSUE TITLE
Tone deaf package name? 

### ISSUE THREAD

**Issue by einarbmag** [seatgeek/fuzzywuzzy#218]
Just checking, are the creators aware of the origin and meaning of the word fuzzy-wuzzy?

**Comment by josegonzalez** [comment]
When we first announced the package, I think the reference we had in mind was the "fuzzy wuzzy was a bear". Cute name for fuzzy string matching. Someone on HN said it was a pejorative in British culture, which we were unaware of, but the name sort of stuck.

I'm happy to rename to something better. The biggest issue is the package naming on pypi, not sure how to handle renames there - can we mark a package as deprecated/redirect?

Any help you can provide here would be great.

**Comment by einarbmag** [comment]
Yeah, I figured it was unintentional. I live in the UK and I wasn't aware
of it either until I spoke about the package to my coworker who's a POC and
he pointed it out.

I don't know what the best thing to do in this kind of situation is -
probably as you say, you should change the name but alias the package with
the old name so as not to break things for existing users. Maybe this can
help : https://pypi.org/project/pypi-alias/, but I would contact pypi for
guidance on how to do it seamlessly.

**Comment by josegonzalez** [comment]
Any ideas for naming?

**Comment by einarbmag** [comment]
Hot fuzz
Fizzy match
Fuzzy fizz

Will keep thinking and ask others

**Comment by rickhanlonii** [comment]
TIL

How about fuzzyfuzzy?

**Comment by KyleHaynes** [comment]
Fuzzy?

**Comment by Eunoia** [comment]
Fussy match:
> (of a person) fastidious about one's needs or requirements; hard to please.

**Comment by markdall** [comment]
Yeek (See: "Little Fuzzy" by H. Beam Piper)

**Comment by josegonzalez** [comment]
If someone wants to make the requisite pull requests for this, I would gladly accept it.

**Comment by freakboy3742** [comment]
@josegonzalez Happy to work up a PR for this - all I need is a decision on the preferred new name.

**Comment by josegonzalez** [comment]
`fuzzy-string-match` maybe?

Note: I no longer work at seatgeek, but can help coordinate changing the repository/making any necessary pipit changes, so long as someone walks me through it :)

**Comment by einarbmag** [comment]
Many good name proposals. 
I vote for fuzzyfuzzy as a minimal name change. fuzzy-string-match is to the point, but maybe a bit generic as there's other fuzzy matching packages out there?

**Comment by josegonzalez** [comment]
Can someone confirm `fuzzyfuzzy` also doesn't mean something odd?

**Comment by songololo** [comment]
`fizzybuzzy`!

**Comment by gw00207** [comment]
this still needs to be changed!

**Comment by Eunoia** [comment]
`fuzzyfuzzy` should be ok. The Parry Gripp song, [fuzzy fuzzy cute cute](https://www.youtube.com/watch?v=X2F4EFYM_MA) has 9.8 MM views, and no negative comments.

Urban dictionary doesn't have any thing negative to say about fuzzy fuzzy, just [fuzzfuzz](https://www.urbandictionary.com/define.php?term=fuzzfuzz) for what two friends named their secret handshake.

It should be culturally safe to rename the package. Caveat: I haven't asked anyone.

**Comment by rickhanlonii** [comment]
@josegonzalez @ekelleyv who has publish rights to make the rename happen?

**Comment by acslater00** [comment]
FWIW I strongly do not think the package name should be changed. This package has 6000 followers on GitHub and numerous language ports, blog posts, stackoverflow answers, etc all under the original name. While I understand that the name has a negative connotation for some people, and that in hindsight a different name could have been chosen to avoid that, I think the ship has long since sailed on this.

If there is a way to alias the package for people who do not want to have to see the original package name in their repos, that would imo be a reasonable concession.

**Comment by josegonzalez** [comment]
I agree with @acslater00. Breaking installations at this point is kinda crappy - yes, I know existing installs won't be broken, but they won't get package updates anymore - but aliasing seems like a good compromise.

@rickhanlonii I no longer have access to any of this stuff - hurray off-boarding! - but the SG infra team should be able to facilitate any repository/pypi renaming, so long as someone figures out the exact steps that should be done so as not to "break" installs.

**Comment by Zonr0** [comment]
Bit of an old bump, but just wanted to chime-in on this one. Big fan of this library but caught a good amount of egg on my face when someone pointed out the term's origins followed by some horrifying googling later.

Any update on the aliasing approach?

**Comment by Drew-dot-A** [comment]
Bumping again for the same reasons as @Zonr0

**Comment by josegonzalez** [comment]
This is being worked on internally.

For a bit more context, we have an internal RFC process. Someone filed an RFC last week as to what to do here, taking into account future maintenance etc. As such, the actual change now has someone owning it, and we should be able to move relatively quickly.

I don't personally have a timeline on this, but the details are being sorted, so it should happen relatively soon.

**Comment by edrandall** [comment]
It's become important to get sorted sooner rather than later and not let it drag on.
